codeforboston/maple
# Extract translatable strings

# Summary

This PR extracts translatable strings from a few of our outstanding components (as determined by our lint rule). I had a little free time and figured I could chip away at a few of these.

# Checklist

- [na] If I've added shared components, I've added a storybook story.
- [na] I've made pages responsive and look good on mobile.

# Screenshots

N/A (No visible changes)

# Known issues

N/A

# Steps to test/reproduce
There should be no visible changes for any of these 
1. Go to the home page and check the HearingsScheduled section
2. Go to the Bill Details page for a bill and check the "Submit Testimony" section
3. Go through the Submit Testimony flow
4. Go to the learn page for the Legislative Process
